src/app/beer_garden/router.py

```diff
@@ -540,34 +540,34 @@ def _determine_target_garden(operation: Operation) -> str:
     if operation.operation_type in ("SYSTEM_RELOAD", "SYSTEM_UPDATE"):
         return _system_id_lookup(operation.args[0])
 
-    elif operation.operation_type == "SYSTEM_DELETE":
+    if operation.operation_type == "SYSTEM_DELETE":
```

Comment:
I guess this is the better way to handle it...

Reply:
Ehh, there's really no right or wrong here. It just needs to be consistent.